src/components/waitlist/WaitlistAlreadyIn.tsx

```diff
@@ -9,7 +9,7 @@ const relayUrls = [
     "wss://eden.nostr.land"
 ];
 
-import { SimplePool } from "nostr-tools";
```

Comment:
This the the fix I am wondering the most about as it produces a inline lint in my nvim stating 
```
const pool = new SimplePool(); ~~This expression is not constructable.
```

Reply:
yeah tbh I've been ignoring nostr-tools stuff because we're going to rip it out when we are done with the waitlist

Reply:
especially as 'nostr-tools' recommends the previous import

[nostr-tools readme file](https://github.com/nbd-wtf/nostr-tools/blob/b9a7f814aaa08a4b1cec705517b664390abd3f69/README.md?plain=1#L136)

wondering how else we can solve this lint

Reply:
going to go ahead and revert this one